include/options/registry.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Options;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Polylang's options registry.
+ *
+ * @since 3.7
+ */
+class Registry {
+	const OPTIONS = array(
+		// URL modifications.
+		Business\Force_Lang::class,
+		Business\Domains::class,
+		Business\Hide_Default::class,
+		Business\Rewrite::class,
+		Business\Redirect_Lang::class,
+		// Detect browser language.
+		Business\Browser::class,
+		// Media.
+		Business\Media_Support::class,
+		// Custom post types and taxonomies.
+		Business\Post_Types::class,
+		Business\Taxonomies::class,
+		// Synchronization.
+		Business\Sync::class,
+		// Internal.
+		Business\Language_Slug::class,
+		Business\Nav_Menus::class,
+		Business\Language_Taxonomies::class,
+		// Read only.
+		Business\First_Activation::class,
+		Business\Previous_Version::class,
+		Business\Version::class,
+	);
+
+	/**
+	 * Registers Polylang's options.
+	 *
+	 * @since 3.7
+	 *
+	 * @param Options $options Instance of the options.
+	 * @return void
+	 */
+	public static function register_options( Options $options ): void {
```

Comment:
```suggestion
	public static function register( Options $options ): void {
```
With namespaces no need to repeat ourselves IMHO.

Reply:
Good idea. Done in [a22ed80](https://github.com/polylang/polylang/pull/1481/commits/a22ed80c00d4facfc7d389f3390e1a31e4d7327c).